change/react-native-windows-2020-05-05-22-41-31-msbuilt.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
FYI, this should be "prerelease". This will otherwise not appear in the changelog, which can get a bit confusing since this does have an effect on the package.

Reply:
Noted!